build.sh

```diff
@@ -131,7 +132,15 @@ if [ "${B_FS}" = "y" ] && [ -d  "${PREFIX_ROOTSKEL}" ]; then
 
 	mkdir -p "${PREFIX_ROOTFS}"
 	cp -a "${PREFIX_ROOTSKEL}/." "${PREFIX_ROOTFS}"
-	mkdir -p "$PREFIX_ROOTFS/"{dev,local,data,mnt,tmp,var}
+	mkdir -p "$PREFIX_ROOTFS/"{dev,etc,local,data,mnt,tmp,var,usr}
+
+	# ROOTFS_OVERLAYS contains colon-separated path
+	(
+		IFS=:
+		for path in $ROOTFS_OVERLAYS; do
+			cp -a "${path}/." "${PREFIX_ROOTFS}"
```

Comment:
only if path exists.
Also - maybe add some log when installing overlay?

Reply:
I am not sure, what you mean. I thought, if `ROOTFS_OVERLAYS` is empty, the `for` loop would not be executed.